src/neptune_sacred/impl/__init__.py

```diff
@@ -72,38 +72,50 @@ class NeptuneObserver(RunObserver):
         ...     per = permutation(iris.target.size)
         ...     iris.data = iris.data[per]
         ...     iris.target = iris.target[per]
-        ...     clf = svm.SVC(C, 'rbf', gamma=gamma)
-        ...     clf.fit(iris.data[:90],
-        ...             iris.target[:90])
-        ...     return clf.score(iris.data[90:],
-        ...                      iris.target[90:])
-
+        ...     clf = svm.SVC(C, "rbf", gamma=gamma)
+        ...     clf.fit(iris.data[:90], iris.target[:90])
+        ...     return clf.score(iris.data[90:], iris.target[90:])
 
-    You may also want to check `sacred integration docs page` and `example experiment page`_.
+    You may also want to check the following:
 
-    .. _sacred integration docs page:
-        https://docs.neptune.ai/integrations-and-supported-tools/model-training/sacred
-    .. _example experiment page:
-        https://app.neptune.ai/prince.canuma/sacred-integration/e/SAC-59/all
+    Sacred integration docs page:
+        https://docs.neptune.ai/integrations/sacred
+    Example run displayed in Neptune:
+        https://app.neptune.ai/o/common/org/sacred-integration/e/SAC-1336/all
```

Comment:
The only issue I see here is that the example dashboard includes hardware consumption metrics, but they didn't seem to be logged for this run.

Not sure how important it is, but we probably want to either capture those or alternatively remove those widgets from the dashboard.

Reply:
@normandy7 updated